src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SEPARATED_VALUES = ",";
+    private static final String OPERATION_NAME = "supply";
 
+    private int [] readFile(String fromFileName) {
```

Comment:
1. Private ones (methods, classes) should be after public ones.
2. This method should only **read** the file. Move calculation to getReport method. Return `List<String>` here